packages/neuron-wallet/src/models/chain/script.ts

```diff
@@ -34,7 +34,17 @@ export default class Script {
   }
 
   public computeHash(): string {
-    return scriptToHash(this.toSDK())
+    const script = this.toSDK()
+    // TODO: should throw error, skip calculate hash if parameter not enough (only in test environment)
+    if (!script || !script.codeHash || !script.hashType) {
+      return ''
+    }
```

Comment:
Why return an empty string here instead of throwing an error?

Reply:
You are right here, I intended to throwing an error (see the TODO), but meet some obstacles:

In test environment, the `.env` file variables can't be read, for example the [SystemScriptInfo.DAO_CODE_HASH](https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/models/system-script-info.ts#L16-L20) here, causing test cases fail to run... 

Anybody can help me to get around with this problem here ? @yanguoyu @devchenyan @Keith-CY

Reply:
updated by [563ef4c](https://github.com/nervosnetwork/neuron/pull/2802/commits/563ef4cb4037d56721d6b1dc2954e427d40518d8)

Reply:
The env for the test case is init at https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/tests/setup.ts. And the local env file will not load because load-env-file is false https://github.com/nervosnetwork/neuron/blob/develop/package.json#L34.